UI/BindingSources/CSHARP/Binding/MainPage.cs

```diff
@@ -0,0 +1,70 @@
+namespace Binding;
+
+public sealed partial class MainPage : Page
+{
+	public MainPage()
+	{
+		this.DataContext(new MainViewModel(), (page, vm)
+			=> page
+			.Background(Theme.Brushes.Background.Default)
+			.Content(
+				new StackPanel()
+					.VerticalAlignment(VerticalAlignment.Center)
+					.HorizontalAlignment(HorizontalAlignment.Center)
+					.Children(
+						new ListView()
+							.ItemsSource(() => vm.Items)
+							.ItemTemplate<Item>(item =>
+								new StackPanel()
+									.Spacing(8)
+									.Orientation(Orientation.Horizontal)
+									.Children(
+										new TextBlock()
+											.Text(() => item.Text)
+											.VerticalAlignment(VerticalAlignment.Center),
+										new Button()
+											.Content("Delete")
+											.CommandParameter(() => item)
+											.Command(x => x.Source(page)
```

Comment:
This could also be
```cs
.Command(x => x.Source(this)
```
You have this below but there isn't any sort of comment explaining it which might leave someone confused as to which they should use. It's important to denote that because we're working within the DataContext of the current object we could substitute either one but if you were using another control with the DataContext you would probably want to reference the alias of the control in the callback method (in this case `page`)